### DISCUSSION TITLE
Ldddns on local network

### DISCUSSION THREAD

**Discussion by morgancasale** [arnested/ldddns#237] [Q&A] [answered]
I was wondering if ldddns supports multicasting on local network (I hope I'm using the right terminologies, I'm pretty new to mdns stuff).
My current setup is composed of two machines connected to the same network;
on machine A I'm running the ldddns service and a docker container (let's say it's called test and it's dns something like test-web-1.local), I can send requests to test-web-1.local from A no problems but when I try to send them from machine B it seem it can't find it.

**Comment by arnested** [comment, marked as answer]
It will only broadcast on your own machine. 

We could make it configurable so you could choose to broadcast on the network you're on.

But I don't think you should still be able to for the traffic from one machine to the container on the other machine. I'm not sure what would be needed to do that. 

Ldddns is only doing the hostname to IP address stuff. No routing or proxying of traffic.
